### PR TITLE
GEOPY-1116: update Pillow as recommended by GitHub advisory

### PR DESCRIPTION
**Pull request by sebhmg**
**GEOPY-1116 - update Pillow as recommended by GitHub advisory**

### CODE REVIEW

File: environments/conda-py-3.10-win-64.lock.yml [hunk 1]
```diff
@@ -299,8 +308,7 @@ dependencies:
   - ipywidgets=7.8.0=pyhd8ed1ab_0
   - ipyfilechooser=0.6.0=pyhd8ed1ab_0
   - pip:
-    - semver === 3.0.2
-    - geoh5py === 0.8.0rc1
+    - geoh5py @ https://github.com/MiraGeoscience/geoh5py/archive/refs/heads/GEOPY-1116.zip
     - lasio === 0.31
```
Comment (sebhmg) [inline environments/conda-py-3.10-win-64.lock.yml:312]: this version does not exit on conda, so has to be on pypi :(

**Review by andrewg-mira — DISMISSED** [review]

File: environments/conda-py-3.10-win-64.lock.yml [hunk 2]
```diff
@@ -299,8 +308,7 @@ dependencies:
...
     - properties === 0.6.1
```
Comment (sebhmg) [inline environments/conda-py-3.10-win-64.lock.yml:313]: properties seems to be on conda, I will fix that one

**Review by domfournier — APPROVED** [review]
Great, less fixed dependencies

**Comment by sebhmg** [comment]
depends on https://github.com/MiraGeoscience/geoh5py/pull/446

**Pull request merged**